hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -1024,8 +1029,9 @@ object HoodieSparkSqlWriter {
 
   private def getHoodieTableConfig(sparkContext: SparkContext,
```

Comment:
javadoc to explain the usage will help demystify it - it's internally more complicated than the name `getHoodieTableConfig` suggests